packages/TEMPLATE/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+# whatever command-line method needs to be used to run this package
```

Comment:
You could add an example here how to run the singularity image from above (using the same `$PLANUTILS_PREFIX` variable used to install it).

Reply:
It's even more direct since it will live next to the script.